spec/tail_spec.rb

```diff
@@ -1,5 +1,6 @@
 require 'filewatch/tail'
 require 'stud/temporary'
+Thread.abort_on_exception = true
```

Comment:
I was wondering how devutils does it but couldn't see it after v brief look. Also googled for best-practice for this but nada.

Will experiment with before after.